### ISSUE TITLE
NTR: REM sleep behavior disorder by history (Lab: neurophysiology: EEG)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9049]
New term request REM sleep behavior disorder by history (segal_220706150439): Dream-enacting behavior; physically act out vivid, often unpleasant dreams with vocal sounds and sudden, often violent arm and leg movements during REM sleep; The affected individual may inflict self-injury or harm others, and is difficult to awaken from such episodes.  Episodes are usually followed by a vivid recollection of a dream that is consistent with the aggressive behavior.  RBD  UMLS: 0751772

**Comment by pnrobinson** [comment]
@MickeySegal

the HPO would code this item as well as #9048 with [Sleep disturbance HP:0002360](https://hpo.jax.org/app/browse/term/HP:0002360), i.e., it would not code the mode of diagnostics. However, the description here goes into some detailed manifestations. We are currently explaning the behavioral branch of the HPO and this might be appropriate there.

**Comment by MickeySegal** [comment]
This needs its own term.  This is like those cat experiments in which a brainstem ablation causes the cats to act out their dreams.  It is a sign of great diagnostic value in distinguishing Parkinson disease from Lewy body disease.  As you will see, we have both history and lab findings for this and I expect you will use the same HPO term for both.

Yesterday I ran a ChatGPS query on findings that fit both Parkinson and Lewy, and it focused only on Parkinson and its other findings, while the correct thing to do is consider the differential diagnosis of Parkinson and Lewy.  This is an example of premature closure in ChatGPS.  We will do better by making the relevant findings computable with HPO / UMLS codes.

**Comment by MickeySegal** [comment]
BTW, this was in the wrong category because it was recently copied from the EEG version of the finding (kalia_091014025726), and this is now in the "Mental status, mental development & sleep" category.

**Comment by MickeySegal** [comment]
My assumption is that the benefit of the sleep EEG is to document that these episodes are happening during REM sleep.  REM sleep, of course, is not abnormal; what is abnormal is the acting out, which can be observed without the EEG.  But the sleep EEG is often done even though its role may be merely to document spousal reports.

But all you need in a finding is to describe the clinical part and we will use that both here and for "REM sleep behavior disorder on sleep EEG" kalia_091014025726

**Comment by pnrobinson** [comment]
Pending resolution of issue https://github.com/obophenotype/human-phenotype-ontology/issues/9048

**Comment by pnrobinson** [comment]
@MickeySegal 
this is [Sleep enactment HP:5200291](https://hpo.jax.org/browse/term/HP:5200291), but note I have changed the primary label to REM sleep behavior disorder
